internal/pkg/v2/utils/http.go

```diff
@@ -7,15 +7,39 @@ package utils
 
 import (
 	"context"
+	"fmt"
 	"net/http"
+	"strconv"
+	"strings"
 
 	"github.com/edgexfoundry/edgex-go/internal/pkg/correlation"
-
 	"github.com/edgexfoundry/go-mod-core-contracts/clients"
+	"github.com/edgexfoundry/go-mod-core-contracts/errors"
 )
 
 func WriteHttpHeader(w http.ResponseWriter, ctx context.Context, statusCode int) {
 	w.Header().Set(clients.CorrelationHeader, correlation.FromContext(ctx))
 	w.Header().Set(clients.ContentType, clients.ContentTypeJSON)
 	w.WriteHeader(statusCode)
 }
+
+func ParseQueryStringToInt(r *http.Request, queryStringKey string) (int, errors.EdgeX) {
+	var result = 0
+	var parsingErr error
+	values, ok := r.URL.Query()[queryStringKey]
+	if ok && len(values) > 0 {
+		result, parsingErr = strconv.Atoi(values[0])
+		if parsingErr != nil {
+			return 0, errors.NewCommonEdgeXWrapper(fmt.Errorf("failed to parse querystring %s's value %s into integer. Error:%s", queryStringKey, values[0], parsingErr.Error()))
+		}
+	}
+	return result, nil
+}
+
+func ParseQueryStringToStrings(r *http.Request, queryStringKey string) (stringArray []string) {
+	values, ok := r.URL.Query()[queryStringKey]
+	if ok && len(values) >= 1 {
+		stringArray = strings.Split(values[0], ",")
+	}
+	return stringArray
```

Comment:
Also need to trim? i.e. could there be spaces that need to be removed.

Reply:
yeah...added `strings.TrimSpace(values[0])` before Split.